src/rwa/components/inputs/number-input.tsx

```diff
@@ -42,9 +42,10 @@ export function RWANumberInput<ControlInputs extends FieldValues>(
             name={name}
             control={control}
             rules={{ required: requiredErrorMessage ?? props.required }}
-            render={({ field: { onChange, value } }) => (
+            render={({ field: { onChange, value, ref } }) => (
```

Comment:
the react number format component has a special `getInputRef` prop for handling the ref of a custom input. Adding this lets react hook form focus the input properly on errors.